app/src/main/res/layout/your_info_home.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Couldn't this be just a single `ConstrantLayout`? Or a set of nested `LinearLayout`s? 🤔